package.json

```diff
@@ -37,18 +37,21 @@
     "@easyops-cn/docusaurus-search-local": "0.44.5",
     "@emotion/react": "11.13.0",
     "@emotion/styled": "11.13.0",
-    "@genesislcap/design-system-configurator": "14.200.0",
-    "@genesislcap/documentation-components": "14.200.0",
-    "@genesislcap/foundation-comms": "14.200.0",
-    "@genesislcap/foundation-entity-management": "14.200.0",
-    "@genesislcap/foundation-filters": "14.200.0",
-    "@genesislcap/foundation-forms": "14.200.0",
-    "@genesislcap/foundation-header": "14.200.0",
-    "@genesislcap/foundation-layout": "14.200.0",
-    "@genesislcap/foundation-login": "14.200.0",
-    "@genesislcap/foundation-reporting": "14.200.0",
-    "@genesislcap/foundation-testing": "14.200.0",
-    "@genesislcap/foundation-zero": "14.200.0",
+    "@genesislcap/design-system-configurator": "14.214.3",
+    "@genesislcap/documentation-components": "14.214.3",
+    "@genesislcap/foundation-comms": "14.214.3",
+    "@genesislcap/foundation-entity-management": "14.214.3",
+    "@genesislcap/foundation-filters": "14.214.3",
+    "@genesislcap/foundation-forms": "14.214.3",
+    "@genesislcap/foundation-header": "14.214.3",
+    "@genesislcap/foundation-layout": "14.214.3",
+    "@genesislcap/foundation-login": "14.214.3",
+    "@genesislcap/foundation-reporting": "14.214.3",
+    "@genesislcap/foundation-testing": "14.214.3",
+    "@genesislcap/foundation-zero": "14.214.3",
+    "@genesislcap/g2plot-chart": "14.214.3",
+    "@genesislcap/grid-pro": "14.214.3",
+    "@genesislcap/grid-tabulator": "14.214.3",
```

Comment:
Nitpick: bumping all the packge versions imply that they've also had their docs pages sycned, is that the case? If not I think we should leave the older package versions as they were and bump and sync them in a different PR.